pkg/render/manager.go

```diff
@@ -705,6 +776,11 @@ func managerClusterRole(managementCluster, managedCluster, openshift bool) *rbac
 				Resources: []string{"serviceaccounts", "namespaces", "nodes", "events"},
 				Verbs:     []string{"list"},
 			},
+			{
```

Comment:
Is it possible to narrow this permission at all? Can we at least limit to exec only for calico/node pods?

Reply:
Sure, I will give it a try. I think it should be fluentd pods.

Reply:
@caseydavenport I think I need to create a role and limit this to a particular namespace. I think ResourceName might require the full name of the pod.

Reply:
Can we do this as a follow up?  Delaying 3.8 testing further is problematic.

Reply:
@asincu are you planning to remove this permission here now that we are granting it via a Role / RoleBinding?